apps/vaporgui/FlowSubtabs.cpp

```diff
@@ -440,26 +442,32 @@ void FlowSeedingSubtab::_updateRake( VAPoR::DataMgr* dataMgr ) {
                                      minExt, 
                                      maxExt, 
                                      axes  );
-    // If there are no valid extents to set the rake with, just return
```

Comment:
There is a lot of complexity in FlowSeedingSubtab::_updateRake(). Some of logic I don't completely understand. But it seems that much the complexity is due to the way FlowParams attempts to handle initialization of the rake via GetRake(). Two thoughts:

1. Can we replace the current rake implementation with a Box params class? This is the right thing to do, and it would  eliminate all of the special handling rake code we're currently struggling with.

2. If (1) is too much of an effort this late in the game, how about simply initializing the rake inside of the FlowParams constructor using the Box params class? This would get away from all of the nans, and complex logic inside of _updateRake.